common/views/AddNewProgress.jsx

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import Transmit from 'react-transmit';
+import {handleRoute} from 'fluxible-router';
+import mui, { RaisedButton, DatePicker, TextField, SelectField } from 'material-ui';
+import pptColors from '../styles/color';
+import pptSpacing from '../styles/spacing';
+
+const { AutoPrefix } = mui.Styles;
+const debug = require('debug')('ppt:AddNewProgress');
+
+class AddNewProgress extends React.Component {
+
+  constructor(props, context) {
+    super(props, context);
+    this.state = {
+      source: '',
+      brief: '',
+      date: undefined,
+      selectedGovernor: undefined,
+      relatedCommitment: undefined
+    };
+  }
+
+  getStyles() {
+    return {
+      root: {
+        paddingTop: pptSpacing.appBarHeight,
+        height: '100%',
+        width: '100%'
+      },
+      section: {
+        width: '100%',
+        maxWidth: 960,
+        margin: 'auto',
+        padding: '0 15px',
+        boxSizing: 'border-box'
+      },
+      middleSection: {
+        width: '100%',
+        marginTop: 24,
+        position: 'relative'
+      },
+      notFullWidth: {
+        display: 'inline-block',
+        width: '50%'
+      }
+    };
+  }
+
+  handleTextInput(stateAttr) {
+    return (e) => {
+      this.setState({
+        [stateAttr]: e.target.value
+      });
+    };
+  }
+
+  handleDateChange(nill, selectedDate) {
+    this.setState({
+      date: selectedDate
+    });
+  }
+
+  handleSubmit() {
+    debug(this.state);
+  }
+
+  render() {
+    const styles = this.getStyles();
+    const governorMenu = [
+      { payload: '1', text: '桃園市政府' },
+      { payload: '2', text: '台中市政府' },
+      { payload: '3', text: '台北市政府' }
+    ];
+    return (
+      <div style={styles.root}>
+        <section style={AutoPrefix.all(styles.section)}>
+          <TextField
+            hintText='請貼上政府公告或新聞的 URL'
+            floatingLabelText='資訊出處'
+            fullWidth={true}
+            inputStyle={{color: pptColors.black}}
+            value={this.state.source}
+            onChange={::this.handleTextInput('source')} />
+          <TextField
+            hintText='請大概節錄出處文章的相關文字'
+            floatingLabelText='資訊摘要'
+            fullWidth={true}
+            inputStyle={{color: pptColors.black}}
+            value={this.state.brief}
+            onChange={::this.handleTextInput('brief')} />
+          <div style={styles.middleSection}>
+            <DatePicker
+              textFieldStyle={{width: '80%'}}
+              style={{...styles.notFullWidth}}
+              hintText="新進度發生時間"
+              showYearSelector={true}
+              onChange={::this.handleDateChange} />
```

Comment:
Damn I am wrong......
What we use here is `DatePicker`, which is a `TextField` + `DatePickerDialog`. However, the properties on `DatePicker` [goes to `TextField`](https://github.com/callemall/material-ui/blob/master/src/date-picker/date-picker.jsx#L83), not `DatePickerDialog`......

Maybe we will have to inherit(?) DatePicker and implement the render function ourselves.